src/bin/agent.rs

```diff
@@ -78,9 +76,15 @@ async fn main() -> anyhow::Result<()> {
         insecure_registries: None,
     };
 
-    let kubeconfig = KubeConfig::from_kubeconfig(&KubeConfigOptions::default())
-        .await
-        .expect("Failed to create Kubernetes Client!");
+    // Bootstrap a kubernetes config, if no valid config is found
+    // This also generates certificates for the webserver the krustlet
+    // runs
+    let kubeconfig = kubelet::bootstrap(
```

Comment:
I looked at the krustlet docs (not code) for this and I can't see the "if no valid config is found" part. Are you sure this is the case?

Reply:
My comment is based on [this documentation](https://github.com/deislabs/krustlet/blob/master/docs/howto/bootstrapping.md#running-krustlet) and my take on what I saw in the [code](https://github.com/stackabletech/krustlet/blob/master/crates/kubelet/src/bootstrapping/mod.rs#L38).

Reply:
Alright, I see. I only looked at the [docs](https://github.com/stackabletech/krustlet/blob/master/crates/kubelet/src/bootstrapping/mod.rs#L22)